src/main/java/bio/terra/service/snapshotbuilder/query/Query.java

```diff
@@ -151,7 +103,55 @@ public String renderSQL(SqlRenderContext context) {
     return "SELECT " + sql;
   }
 
-  public SourceVariable getPrimaryTable() {
-    return tables.stream().filter(SourceVariable::isPrimary).findFirst().orElseThrow();
+  public Table getPrimaryTable() {
```

Comment:
Why is this table and not sourceVariable?

Reply:
Shelby renamed from `FieldVariable` to `SourceVariable`. Before merging with Shelby, I renamed the `FieldVariable` to `Table` because the tables were no longer going to be a general `FieldVariable` but a more specific `Table` type, which are `Concept`, `ConceptAncestor` ... `Person`. So, we don't want to return a general `SourceVariable` because its encapsulated inside of `Table`.

Reply:
We can ask a `Concept` object, "hey, are you primary" like this: 
```
Concept concept = Concept.asPrimary() // creates a concept object
boolean isPrimary = concept.isPrimary(); 
```